README.md

```diff
@@ -1,194 +1,111 @@
-|`final_eval_batch_size` |Final model evaluation parameter for MMLU. Batch size for evaluation. Valid values are a positive integer or 'auto' to select the largest batch size that will fit in memory.|
-|`final_eval_merge_system_user_message` |Final model evaluation parameter for MT Bench Branch. Boolean indicating whether to merge system and user messages (required for Mistral based judges)|
-|`k8s_storage_class_name` |A Kubernetes StorageClass name for persistent volumes. Selected StorageClass must support RWX PersistentVolumes.|
-
-
-### Customize the Pipeline
-
-The `pipeline.yaml` provided in this repo will always represent the most up to date version of the pipeline as our team continues to improve upon it as well as keep it in line with the InstructLab CLI. However, if you are a contributor or simply want to experiment with making custom changes to the pipeline that can be done by simply editing and "compiling" the `pipeline.py` file provided in this repo.
-
-The pipeline yaml is defined by `pipeline.py` file and then converted into an intermediate representation yaml that Data Science Pipelines expects via the KubeFlow Pipelines python SDK. If you want to customize the pipeline in anyway, you can update `pipeline.py`, run the below make command and then upload the pipeline to your Data Science Pipeline instance similar to how we showed [above](#upload-the-pipeline).
+Deploy it in your DS project:
 
 ```bash
-make pipeline
+oc -n <data-science-project-name/namespace> oc apply -f oci_output_push_secret.yaml
 ```
 
-## Developer setup
+Note the `metadata.name` of this secret, you will need this when filling out the InstructLab pipeline parameters.
 
-To collaborate on this repository, please follow these steps:
+## Run the Pipeline
 
-1. Install [uv](https://docs.astral.sh/uv/getting-started/installation/)
-2. Run following commands to prepare your local environment
-    ```bash
-    uv sync
-    source .venv/bin/activate
-    ```
+Now that all the cluster requirements have been set up, we are ready to upload and run our InstructLab pipeline!
 
-## Adding/Updating dependencies
+Starting the InstructLab pipeline run is the same as starting any other run using Data Science Pipelines.
 
-When updating python package dependencies in `pyproject.toml`, regenerate [requirements.txt](requirements.txt):
+The instructions for how to create a typical run can be found [here][DSP Run Docs]. For the pipeline selection step, you will be able to select the `InstructLab` pipeline from the pipeline list when creating a run.
 
-```
-uv pip compile pyproject.toml --generate-hashes > requirements.txt
-```
+You can find a description of each Parameter once you go to the run creation page. Some parameters already have defaults, while others do not, and some are optional. Please carefully read the description of each to better customize the pipeline to meet your needs.
```

Comment:
```suggestion
You can find a description of each Parameter once you go to the run creation page. Some parameters already have defaults, while others do not, and some are optional. Please carefully read the description of each to better customize the pipeline to meet your needs. 

Beware there is a [bug](https://issues.redhat.com/browse/RHOAIENG-22522) in RHOAI 2.19 where empty parameters do not appear when duplicating a run, so during your testing you might have to create a new pipeline runs instead of duplicating them.
```